pkg/golinters/cyclop.go

```diff
@@ -0,0 +1,21 @@
+package golinters
+
+import (
+	"github.com/bkielbasa/cyclop/pkg/analyzer"
+	"golang.org/x/tools/go/analysis"
+
+	"github.com/golangci/golangci-lint/pkg/golinters/goanalysis"
+)
+
+const cyclopName = "cyclop"
+
+func NewCyclop() *goanalysis.Linter {
+	return goanalysis.NewLinter(
+		cyclopName,
+		"checks function and package cyclomatic complexity",
+		[]*analysis.Analyzer{
+			analyzer.NewAnalyzer(),
+		},
```

Comment:
```suggestion
		[]*analysis.Analyzer{analyzer.NewAnalyzer()},
```

Reply:
Quite opinionated suggestion, do we have a standard for this? Seems like the authors pattern is used a lot, f.ex. in [gomnd](https://github.com/golangci/golangci-lint/blob/master/pkg/golinters/gomnd.go#L12-L14), [nlreturn](https://github.com/golangci/golangci-lint/blob/master/pkg/golinters/nlreturn.go#L14-L16), [tparallel](https://github.com/golangci/golangci-lint/blob/master/pkg/golinters/tparallel.go) and a bunch of other places in the source code. Not sure we should request changes for these kind of things?